shared/test/Datadog.Trace.ClrProfiler.Native.Tests/string_test.cpp

```diff
@@ -17,16 +17,18 @@ TEST(string, ToString)
 
     EXPECT_TRUE("42" == ToString(42));
 
+#ifndef LINUX
```

Comment:
Should we prefer positive tests like:
`   #ifdef WINDOWS`
instead of:
`   #ifndef LINUX`
?

Reply:
as long as it compiles... :)